pep-0655.rst

```diff
@@ -14,16 +14,21 @@ Post-History: 31-Jan-2021, 11-Feb-2021, 20-Feb-2021, 26-Feb-2021, 17-Jan-2022, 2
 Abstract
 ========
 
-:pep:`589` defines syntax
-for declaring a TypedDict with all required keys and syntax for defining
+:pep:`589` defines notation
+for declaring a TypedDict with all required keys and notation for defining
 a TypedDict with :pep:`all potentially-missing keys <589#totality>` however it
-does not provide any syntax to declare some keys as required and others
-as potentially-missing. This PEP introduces two new syntaxes:
+does not provide any notation to declare some keys as required and others
+as potentially-missing. This PEP introduces two new notations:
```

Comment:
```suggestion
as potentially-missing. Specifically, this PEP introduces:
```

Fix grammar/usage error, awkward repetition and unclear semantic flow.

Reply:
This suggestion appears to place the emphasis in an unusual location. Ignoring.